src/main/java/thePackmaster/packs/MariDebuffPack.java

```diff
@@ -0,0 +1,37 @@
+package thePackmaster.packs;
+
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.localization.UIStrings;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.cards.maridebuffpack.*;
+
+import java.util.ArrayList;
+
+public class MariDebuffPack extends AbstractCardPack {
+
+    public static final String ID = SpireAnniversary5Mod.makeID("MariDebuffPack");
+    private static final UIStrings UI_STRINGS = CardCrawlGame.languagePack.getUIString(ID);
+    public static final String NAME = UI_STRINGS.TEXT[0];
+    public static final String DESC = UI_STRINGS.TEXT[1];
+    public static final String AUTHOR = UI_STRINGS.TEXT[2];
+
+    public MariDebuffPack() {
+        super(ID, NAME, DESC, AUTHOR, new PackSummary(4, 2, 5, 3, 4, PackSummary.Tags.Debuffs));
```

Comment:
What's the thought behind the 5 in support? I don't see much card manip, energy gain, etc in this pack.
I'd argue for reducing scaling to 3 as well, your only scaling is the strength gain and a single card scale in Flying car.